packages/common/src/modules/register.ts

```diff
@@ -51,6 +53,7 @@ container.bind(FavoriteService).toSelf();
 container.bind(GenericEntitlementService).toSelf();
 container.bind(ApiService).toSelf();
 container.bind(SettingsService).toSelf();
+container.bind(AccessService).toSelf();
```

Comment:
I think it might be more intuitive to only bind AccessService and Controller if the middleware url is present, instead of checking the middleware url in each method call.

Reply:
That's something that has been bothering me as well, but since we moved the url in the .ini, we don't have that info on this point where we bind the services/controllers as that's a part of the Settings service.